### DISCUSSION TITLE
Can't get the autoshow of Onboard Keyboard working

### DISCUSSION THREAD

**Discussion by WebfinityDesign** [photobooth-app/photobooth-app#325] [General]
I'm trying to get the autoshow function of Onboard Keyboard working, but I can't. Ticking the "autoshow" box doesn't work. Can someone point me in the right direction?

**Comment by mgineer85** [comment]
I am sorry, this is just for documentation available. I find these onscreen keyboards work very bad in general.
I think you are better served with a Bluetooth keyboard.

**Reply by WebfinityDesign** [reply]
Ok. Good to know. I will look out for a BT keyboard.
One question: what does the switch **Keyboard Input Enabled** do under **Hardware Input/Output Config**?
Does it have something to do with the keyboard triggers in the actions sections?

**Reply by mgineer85** [reply]
Yes , it will enable the listener in the ui for the configured keys globally.
